kpm/lib/kpm/account.rb

```diff
@@ -72,7 +72,7 @@ class Account
     DEFAULT_DELIMITER = "|"
 
     def initialize(config_file = nil, killbill_api_credentials = nil, killbill_credentials = nil, killbill_url = nil,
-                   database_name = nil, database_credentials = nil, data_delimiter = nil, logger = nil)
+                   database_name = nil, database_credentials = nil, host = nil, data_delimiter = nil, logger = nil)
```

Comment:
`database_host` instead of `host` to make it more obvious and consistent (same below and in `Database`).